mayastor/src/target/iscsi.rs

```diff
@@ -233,6 +233,9 @@ fn share_as_iscsi_target(
 /// Export given bdev over iscsi. That involves creating iscsi target and
 /// adding the bdev as LUN to it.
 pub fn share(bdev_name: &str, bdev: &Bdev, side: Side) -> Result<String> {
+    if bdev.is_claimed() {
+        return Err(Error::CreateTarget {});
```

Comment:
Should this have a message like the nvmf?

Reply:
Done, for consistency with nvmf.